sdk/identity/Azure.Identity/src/Azure.Identity.csproj

```diff
@@ -25,7 +25,7 @@
     <PackageReference Include="System.Text.Json" />
     <PackageReference Include="System.Threading.Tasks.Extensions" />
     <PackageReference Include="Microsoft.Identity.Client" />
-    
+    <PackageReference Include="System.Security.Cryptography.ProtectedData" />
```

Comment:
Just so you are aware this only works on windows (https://github.com/dotnet/corefx/issues/22510) so if we need it for cross-platform then we will need another solution.

Reply:
Yes I'm aware of this.  We're only supporting SSO on windows since the only participating App is VS on windows so far.  They are possibly going to use a different approach on different platforms so until that's decided the code assumes that we're executing on windows.